daemons/dss-scalability-test/requirements.txt

```diff
@@ -1,2 +1,4 @@
 hca >= 3.2.0
+Faker >= 0.8.11
+rstr >=2.2.6
```

Comment:
I think this should be done at the *top-level* `requirements.txt` and/or `.in` files.

When the JSON generator was recently moved from `tests` to `dss` its dependencies should have been moved from `requirements-dev.txt` to `requirements.txt`, too.

Reply:
We should not include these libraries in the general daemon package.

I think this duplication is fine for now. If you really want to avoid duplication, let's split the json generator requirements into a separate requirements file and include it from requirements-dev and from daemons/dss-scalability-test/requirements.

Reply:
Alternatively, the dss-scalability-test daemon could simply symlink to requirements-dev.txt instead of requirements.txt (I hope all the stuff we have in requirements-dev is packageable).

Reply:
I vote for just symlinking. The requirements handling is already unwieldy as it is. The amount of developer time & money spent with managing multiple separate dependency lists by far outweighs the space and bandwidth savings they accomplish.